src/ripple/app/ledger/impl/InboundLedger.cpp

```diff
@@ -466,8 +471,8 @@ void InboundLedger::trigger (Peer::ptr const& peer)
                 Message::pointer packet (std::make_shared <Message> (
                     tmBH, protocol::mtGET_OBJECTS));
                 {
-                    for (PeerSetMap::iterator it = mPeers.begin (), end = mPeers.end ();
-                            it != end; ++it)
+                    for (auto it = mPeers.begin (), end = mPeers .end ();
```

Comment:
I'd rather see a [FOLD] commit that changes this to:

    for (auto const& it : mPeers)